docs_espressif/en/buildproject.rst

```diff
@@ -1,2 +1,21 @@
+.. _build the project:
+
 Build the Project
 ===============================
+
+Now to build the project, select menu **View**, **Command Palette**, type **ESP-IDF: Build your Project** command. 
```

Comment:
Better approach would be - 

Now to build the project, select menu **View** > **Command Palette** and type **ESP-IDF: Build your Project** command.

Reply:
Follow the same approach everywhere.